addresses.json

```diff
@@ -235,6 +235,16 @@
         "network_name": "tardigrade",
         "rest": "https://lcd.tgrade.posthuman.digital"
     },
+    {
+        "chain_name": "terra",
+        "network_name": "columbus-5", 
```

Comment:
It could be useful to print out the colloquial chain name instead of the exact chain ID